### ISSUE TITLE
truncated tty name

### ISSUE THREAD

**Issue by akhenakh** [f4exb/serialDV#1]
Trying to use serialDV on OSX where some tty name can be very long:

```
crw-rw-rw-  1 root  wheel   18,   6 Jul 21 12:21 /dev/tty.usbmodem1411
```

Got the error

```
SerialDataController::open: Cannot open device - /dev/tty.usbmodeFailed to open DV serial device /dev/tty.usbmode. Aborting
```

Using dtruss we can see the name is really truncated:

```
open("/dev/tty.usbmode\0", 0x20006, 0x0)         = -1 Err#2
write_nocancel(0x2, "SerialDataController::open: Cannot open device - /dev/tty.usbmode\0", 0x41)         = 65 0
write_nocancel(0x2, "Failed to open DV serial device /dev/tty.usbmode. Aborting\n\0", 0x3B)      = 59 0
```

**Comment by f4exb** [comment]
This is the dvtest program that is affected. The open method of the library takes a std::string for the tty name so its length is unlimited. Fix on the way for dvtest ...

**Comment by f4exb** [comment]
Fixed in version v1.0.1
